service/utility.go

```diff
@@ -44,9 +44,9 @@ var (
 	//https://godoc.org/github.com/google/uuid
 	fuidRegex = regexp.MustCompile("^[[:digit:]a-f]{8}-[[:digit:]a-f]{4}-[[:digit:]a-f]{4}-[[:digit:]a-f]{4}-[[:digit:]a-f]{12}$")
 
-	imageRegex = regexp.MustCompile("^.*(jpg|jpeg|png|bmp|tif|gif|tiff).*$")
-	audioRegex = regexp.MustCompile("^.*(wav|wma|ogg|m4a|mp3).*$")
-	videoRegex = regexp.MustCompile("^.*(flv|wmv|mov|avi|mp4).*$")
+	imageRegex = regexp.MustCompile("^.*\\.(jpg|jpeg|png|bmp|tif|gif|tiff)$")
+	audioRegex = regexp.MustCompile("^.*\\.(wav|wma|ogg|m4a|mp3)$")
+	videoRegex = regexp.MustCompile("^.*\\.(flv|wmv|mov|avi|mp4)$")
```

Comment:
For python, I need only one back \ before the . Might be different for go lang, but it could be 
"^.*\.(flv|wmv|mov|avi|mp4)$"?

Reply:
Sorry I meant : "^.*\.(flv|wmv|mov|avi|mp4)$"?

Reply:
`.` means any character, unless you escaped it